app/celer-sim/Runner.cc

```diff
@@ -350,6 +350,7 @@ void Runner::build_transporter_input(RunnerInput const& inp)
     transporter_input_->num_track_slots
         = ceil_div(inp.num_track_slots, core_params_->max_streams());
     transporter_input_->max_steps = inp.max_steps;
+    transporter_input_->store_track_counts = inp.write_track_counts;
```

Comment:
Store in one place, write in another? 😅 I get it though. I think for 1.0 we'll revisit these options anyway.